apps/site/components/SectionItem/section-item.module.css

```diff
@@ -56,6 +82,15 @@
   max-height: 21rem;
 }
 
+.sectionItem[data-doc-section-item-full-width] > div > * { width: 100%; }
+
+.sectionItem[data-doc-section-item-full-width] [data-fs-product-shelf-items] { width: 100%; }
+
+.sectionItem[data-doc-section-item-zoom-out="true"] > div > * {
+  width: 100%;
+  zoom: 60%;
```

Comment:
As the mdn: [transform: scale()](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scale) should be used instead of this property. Do you think does it worthy?

Reply:
I've tested with the `transform: scale()`, but it doesn't work the same way, for some reason, the scale property doesn't scale all the elements in the same proportion.

|Scale|Zoom|
|-|-|
|<img width="354" alt="image" src="https://github.com/vtex/faststore/assets/11613011/b1c49e28-c5b4-403b-9632-a104130883f9">|<img width="354" alt="image" src="https://github.com/vtex/faststore/assets/11613011/928d274d-e0bf-4128-9486-cc8d0fe9f695">|